modules/ppcp-blocks/src/BlocksModule.php

```diff
@@ -88,6 +88,28 @@ static function () use ( $c ) {
 				$endpoint->handle_request();
 			}
 		);
+
+		// Enqueue frontend scripts.
+		add_action(
+			'wp_enqueue_scripts',
+			static function () use ( $c ) {
+				if ( ! has_block( 'woocommerce/checkout' ) && ! has_block( 'woocommerce/cart' ) ) {
```

Comment:
This may not load in editor (if block not added yet).

If we want to fix that, here is how I check it in the messaging branch https://github.com/woocommerce/woocommerce-paypal-payments/pull/1897/commits/4e34332c0bb476866106998dd097619c59692cf2

Reply:
I don't think we should load in the block editor right now as the preview there currently isn't working, maybe address it in the context of that fix:
![image](https://github.com/woocommerce/woocommerce-paypal-payments/assets/45217709/a8962f4d-58cd-4b8a-a59e-f9a1ab43e9eb)